src/com/archos/filecorelibrary/samba/SambaConfiguration.java

```diff
@@ -38,7 +38,7 @@
 
 public class SambaConfiguration {
 
-	protected static final String configFile = Environment.getExternalStorageDirectory()+"/Android/data/com.archos.mediacenter/smb/credentials.conf";
+	protected static final String configFile = Environment.getExternalStorageDirectory()+"/Android/data/org.courville.nova/files/smb/credentials.conf";
```

Comment:
only issue I have with this change is that there is no migration path meaning that a user upgrading will loose his smb credentials.